pkg/skaffold/constants/constants.go

```diff
@@ -73,6 +73,9 @@ var (
 
 	// BuildContext is the absolute path to a directory this artifact is meant to be built from for custom artifacts
 	BuildContext = "BUILD_CONTEXT"
+
+	// Kubecontext is the expected kubecontext to build an artifact with a custom build script on cluster
+	Kubecontext = "KUBECONTEXT"
```

Comment:
nit: KubeContext shd match the "BUILD_CONTEXT" convention
```suggestion
	KubeContext = "KUBE_CONTEXT"
```